src/Lumi/Components/Details.purs

```diff
@@ -0,0 +1,52 @@
+module Lumi.Components.Details where
+
+import Prelude
+
+import Effect.Unsafe (unsafePerformEffect)
+import JSS (JSS, jss)
+import Lumi.Components.Icon (icon_)
+import Lumi.Components.Text (body_)
+import React.Basic.DOM as R
+import React.Basic.Hooks (JSX, component, element)
+
+type DetailsProps =
+  { summary :: JSX
+  , expanded :: JSX
+  , defaultOpen :: Boolean
+  }
+
+details :: DetailsProps -> JSX
+details = element $ unsafePerformEffect do
+  component "Details" \props ->
```

Comment:
I'm not sure we need this to be an actual component if it's just a `<details>` tag without any internal state.